selfdrive/car/tesla/radar_interface.py

```diff
@@ -15,6 +15,8 @@
 RADAR_B_MSGS = list(range(0x311, 0x36F, 3))
 OBJECT_MIN_PROBABILITY = 20.
 CLASS_MIN_PROBABILITY = 20.
+RADAR_MESSAGE_FREQUENCY = 0.050 * 1e9 #time in ns, radar sends data at 0.06 s
+VALID_THRESHOLD = 10
```

Comment:
Consider specifying what this threshold represents.
Ex: `VALID_MESSAGE_COUNT_THRESHOLD`

Reply:
done